psl-core/src/main/java/org/linqs/psl/application/inference/LazyMPEInference.java

```diff
@@ -20,6 +20,7 @@
 import org.linqs.psl.config.Config;
```

Comment:
Unused import.

Reply:
It is used for the constants in the constructor.